src/models.py

```diff
@@ -157,12 +124,13 @@ def build_model(args, vocab, pretrained_embs, tasks):
         log.info("No shared encoder (just using word embeddings)!")
     else:
         assert_for_log(False, "No valid sentence encoder specified.")
+    return sent_encoder, d_sent
 
-    d_sent += args.skip_embs * d_emb
-
-    # Build model and classifiers
-    model = MultiTaskModel(args, sent_encoder, vocab)
-
+def build_task_modules(args, tasks, model, d_sent, d_emb, embedder, vocab):
```

Comment:
nit: while we're moving things, can we either move this to below `build_model()`, or move `build_task_specific_modules()` to above this function so the control flow doesn't jump around so much in this file?